keyboards/handwired/yamsek/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2023 Johannes Schneider (@JohSchneider) */
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
```

Comment:
Can be inferred by the info.json info.
```suggestion
```

Reply:
Actually this suggestion was wrong, because the board uses a custom matrix, so there is nothing to infer the matrix size from (unless maybe the old code had some fake `MATRIX_ROW_PINS` and `MATRIX_COL_PINS` defined somewhere).

However, the matrix size may be specified in `info.json` (`matrix_size.cols` and `matrix_size.rows`).